Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM python:3.11.9-slim-bullseye as base
+RUN apt-get update && apt-get install curl -y
+RUN export POETRY_HOME=/usr/local; curl -sSL https://install.python-poetry.org | python3.11 -
+EXPOSE 8000
+WORKDIR /opt/todoapp
+COPY ["poetry.lock", "poetry.toml", "pyproject.toml", "./"]
```

Comment:
First time I've ever seen the exec form syntax for a COPY command - not sure it makes any real difference here, but worth noting this isn't required for copying multiple files in case that's the expectation, the below should work equally:
```Dockerfile
COPY poetry.lock poetry.toml pyproject.toml ./
```